spark-publish/test/test.js

```diff
@@ -79,6 +79,66 @@ describe('integration', () => {
   })
 
   it('publishes', async () => {
+    await client.query('DELETE FROM measurements')
+
+    const measurementRecorded = {
+      sparkVersion: '1.2.3',
+      zinniaVersion: '0.5.6',
+      cid: 'bafytest',
+      providerAddress: '/dns4/localhost/tcp/8080',
+      protocol: 'graphsync',
+      // We are recording wallet address in Filecoin format (f4...)
+      walletAddress: 'f410ftgmzttyqi3ti4nxbvixa4byql3o5d4eo3jtc43i',
```

Comment:
could you please also add a test for testnet addresses?

Reply:
Done - see [fc711f1](https://github.com/filecoin-station/spark-api/pull/89/commits/fc711f16a6f37eb914719bf23437fb07c8216899)